src/Middleware/HoneybadgerContext.php

```diff
@@ -55,11 +81,15 @@ private function setRouteActionContext()
 
     private function setUserContext($request)
     {
-        if ($request->user()) {
-            $this->honeybadger->context(
-                'user_id',
-                $request->user()->getAuthIdentifier()
-            );
+        try {
+            if ($request->user()) {
+                $this->honeybadger->context(
+                    'user_id',
+                    $request->user()->getAuthIdentifier()
+                );
+            }
+        } catch (\InvalidArgumentException $e) {
+            // swallow
```

Comment:
This helps avoid issues in Lumen where the auth driver and service provider are not registered by default.